github/libp2p.yml

```diff
@@ -6499,6 +6499,13 @@ repositories:
     default_branch: main
     delete_branch_on_merge: true
     description: The JavaScript Implementation of libp2p networking stack.
+    files:
+      .github/pull_request_template.md:
+        content: .github/js_pull_request_template.md
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```

Comment:
Not sure what this is doing in my PR?

Reply:
@achingbrain @galargh do you why automation pushed these changes to an unrelated pull request?

Reply:
The automation runs https://github.com/libp2p/github-mgmt/blob/master/scripts/src/actions/fix-yaml-config.ts after every push to the default branch and on every PR.

The reason why unrelated changes were pushed to the PR was that https://github.com/libp2p/github-mgmt/pull/197 has not been merged. The timeline would be, 1. the sync workflow discovers divergence between what's in this repo and the actual GitHub state, 2. the repo gets updated, 3. the PR with fixes is created but not merged, 4. this PR is created, 5. the fixes are also proposed in this PR because the PR with fixes was not merged yet.

Another thing to note is that once a fix PR is merged, then all the existing PRs are rebased which would have got rid of the unrelated changes.

It is a bit clunky at the moment. I'd like to package the workflows that do the actual work in this repo into nice reusable actions so it'd be easier for users to reason about and customise what happens when. There's no timeline for that work though.
 
Another issue, the commits with fixes are associated with PR owner at the moment which shouldn't be the case as they are bot commits. I created an issue for that here - https://github.com/ipdxco/github-as-code/issues/110